client/src/components/App/App.jsx

```diff
@@ -1,41 +1,99 @@
-        <Form />
-      </Box>
-      <Box mt={2} ml={6} display="flex" flexDirection="column" alignItems="center">
-        <img src={illustration} alt="Woman receiving a vaccine" style={{maxWidth: '750px'}}/>
-        <Link
-          href="https://www.vecteezy.com/free-vector/epidemic-prevention"
-          color="secondary"
-          target="_blank"
-          rel="noopener"
-        >
-          Epidemic Prevention Vectors by Vecteezy
-        </Link>
-      </Box>
-    </Box>
 
+    <Router>    
+      <HeroBar />
+
+      <Switch>
+        <Route exact path="/">
+          <Box
+            display="flex"
+            flexDirection="row"
+            justifyContent="center"
+          >
+            <Box mt={11} mr={4} maxWidth="530px">
+              <Form />
+            </Box>
+            <Box mt={2} ml={6} display="flex" flexDirection="column" alignItems="center">
+              <img src={illustration} alt="Woman receiving a vaccine" style={{maxWidth: '750px'}}/>
+              <Link
+                href="https://www.vecteezy.com/free-vector/epidemic-prevention"
+                color="secondary"
+                target="_blank"
+                rel="noopener"
+              >
+                Epidemic Prevention Vectors by Vecteezy
+              </Link>
+            </Box>
+          </Box>
+        </Route>
+
+        <Route exact path="/data-found">
+          <BackButton />
+          <Box
+            display="flex"
+            flexDirection="row"
+            justifyContent="center"
+            mt={20}
+            mb={20}
+          >
+            SMART HEALTH CARD WILL GO HERE
+          </Box>
+          <Box display="flex" flexDirection="row" justifyContent="center">
+            <DownloadButton>
+              Download SMART&reg; Health Card
+            </DownloadButton>
+            <DownloadButton>
+              Download Printable PDF
+            </DownloadButton>
+          </Box>
+          <Box display="flex" flexDirection="row" justifyContent="center" mt={2} mx={5}>
+            <FactList />
+          </Box>
+          <Box display="flex" flexDirection="row" justifyContent="center" mt={2} mb={2} mx={5}>
+            <Box display="flex" flexDirection="column">
+              <Typography variant="body2">
+                <p>
+                  This vaccination record is powered by SMART&reg; and works with any app or service
+                  where SMART&reg; Health Cards are accepted. For more information, please visit 
+                  <Link color="secondary" to="https://smarthealth.cards/">https://smarthealth.cards</Link>
+                </p>
+                <p>
+                  The IAL number on this document describes the way that your identity information
+                  was collected at the clinic. You can find more information about IAL encoding 
+                  here: <Link color="secondary" to="https://vci.org/ig/vaccination-and-testing">https://vci.org/ig/vaccination-and-testing</Link>
```

Comment:
```suggestion
                  here: <Link color="secondary" to="https://vci.org/ig/vaccination-and-testing">https://vci.org/ig/vaccination-and-testing</Link>.
```
Add period at end of sentence.